packages/server-wallet/src/protocols/__test__/close-channel.test.ts

```diff
@@ -1,70 +1,151 @@
-import {simpleEthAllocation, BN, State} from '@statechannels/wallet-core';
-
-import {protocol} from '../close-channel';
-import {alice, bob} from '../../wallet/__test__/fixtures/participants';
-import {SignState} from '../actions';
+import {testKnex as knex} from '../../../jest/knex-setup-teardown';
+import {MockChainService} from '../../chain-service';
+import {defaultTestConfig} from '../../config';
+import {createLogger} from '../../logger';
 import {ChainServiceRequest, requestTimeout} from '../../models/chain-service-request';
-import {channel} from '../../models/__test__/fixtures/channel';
+import {DBCloseChannelObjective} from '../../models/objective';
+import {Store} from '../../wallet/store';
+import {WalletResponse} from '../../wallet/wallet-response';
+import {TestChannel} from '../../wallet/__test__/fixtures/test-channel';
+import {ChannelCloser} from '../close-channel';
 
-import {applicationProtocolState} from './fixtures/protocol-state';
+const FINAL = 10; // this will be A's state to sign
+const testChan = TestChannel.create({aBal: 5, bBal: 5, finalFrom: FINAL});
 
-const outcome = simpleEthAllocation([{amount: BN.from(5), destination: alice().destination}]);
+let store: Store;
 
-const participants = [alice(), bob()];
-const postFundState = {outcome, turnNum: 3, participants};
-const closingState = {outcome, turnNum: 4, isFinal: true, participants};
+beforeEach(async () => {
+  store = new Store(
+    knex,
+    defaultTestConfig().metricsConfiguration.timingMetrics,
+    defaultTestConfig().skipEvmValidation,
+    '0'
+  );
+  await store.dbAdmin().truncateDB();
+});
 
-const runningState = {outcome, turnNum: 7, participants};
-const closingState2 = {outcome, turnNum: 8, isFinal: true, participants};
+afterEach(async () => await store.dbAdmin().truncateDB());
```

Comment:
Do you need to truncate before _and_ after?

Reply:
Not sure. I'll change the `afterEach` to an `afterAll`, so it does it before each test and once after all the tests.